readme.md

```diff
@@ -167,171 +180,158 @@ console.log(ulChildren);
+  - set: removes all of the elements children and replaces with whatever newContent is
 
-## Altering DOM Elements (5 min)
 
-- [.textContent](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent)
-- .innerHTML
-  this is used as both a getter and setter method
+  *Get Example*
+  ```javascript
+  var getText = $( "ul" ).text();
+  console.log(getText);
+});
+```
+*Set Example*
+```javascript
+$( "ul" ).text("<li>blue</li>");
+```
 
-  Getter:
-  ```js
-  document.querySelector("someElement").innerHTML
-  ```
 
-  Setter:
-  ```js
-  document.querySelector("someElement").innerHTML = "the text that will be inserted for that element"
-  ```
+`.attr()`
+- get: returns the value of an attribute for the first element in the set of matched elements
+- set: set the value of an element attribute
 
-- .setAttribute(name, value);
-  Ex:
-  ```js
-  document.querySelectorAll("img").setAttribute("src", "www.someImageURL.com")
+  *Get Example*
+  ```javascript
+  var getAttr = $('html').attr('lang')
+  console.log(getAttr);
+  });
   ```
 
-  > sets all images to have the source of `"www.someImageURL.com"`
-
-- .id
-  sets or gets an ID for a DOM element
+  *Set Example*
+  ```javascript
+  var setAttr = $('img').attr('src','http://www.clipartlord.com/wp-content/uploads/2014/05/unicorn4.png')
+});
+```
 
-- .style
-  Getter:
-  ```js
-  document.querySelector("someElement").style.background
-  ```
+####You Do: Document Dive and examples (15 minutes)
 
-  Setter:
-  ```js
-  document.querySelector("someElement").style.background = "blue"
-  ```
+For the remaining two methods, find a partner, research, and provide an example of getting and setting for each. Be prepared to share your findings with the class. For '.val()', create an input tag in your HTML for the method to work effectively.
 
-> Note: these are just some snippets, read the full [MDN DOM documentation](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model) for more examples
+```html
+<input type="text" class="search" placeholder="some text here ....">
+```
 
-## You do: Logo hijack (15 min)
+- `.css()`
+[documentation](http://api.jquery.com/css/)
 
-1. Open up www.google.com in Chrome or Firefox, and open up the console.
-- find an image url for the yahoo logo
-- Store the url to the Yahoo logo in a variable.
-- Find the Google logo using JS and store it in a variable.
-- Modify the source of the logo IMG so that it's a Yahoo logo instead.
-- Find the Google search button and store it in a variable.
-- Modify the text of the button so that it says "Yahooo!" instead.
+- `.val()`
+[documentation](http://api.jquery.com/val/)
 
-Bonus: Add a new element between the image and the search textbox, telling the world that "Yahoo is the new Google".
 
-## Creating/Removing DOM Elements (1 min)
+## Adding content
 
-- [document.createElement](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement)
```

Comment:
think it would be good to show them how to create an element before appending one:

```js
var newEl = $("<div>this is a new div!</div>")
$("body").append(newEl)
```